analyzer/testdata/src/rules/main.go

```diff
@@ -0,0 +1,17 @@
+// This package tests rules specified in `rules.go` of ruleguard itself.
```

Comment:
Can we reuse another rules and not create new?

Reply:
What do you mean? This file exists because there are no tests for the root `rules.go` yet. However, it's good to test them.